src/main/kotlin/no/nav/syfo/dialogmote/domain/DialogmotedeltakerArbeidstakerVarsel.kt

```diff
@@ -17,8 +17,20 @@ data class DialogmotedeltakerArbeidstakerVarsel(
     val status: String,
     val lestDato: LocalDateTime?,
     val fritekst: String,
+    val document: List<DocumentComponentDTO>,
 )
 
+data class DocumentComponentDTO(
```

Comment:
Flyttet DocumentComponentDTO hit siden den blir en del av modellen (og ikke bare api-modellen). Usikker på om det er riktig å la "DTO" være en del av navnet, men tror kanskje det er greit for å markere at denne persisteres as-is som en json.